types/header/header.go

```diff
@@ -1,6 +1,11 @@
 package header
```

Comment:
if you rely on blob module alone for both check data availability and retrieve blocks, then you can remove the header related types? i am aware that we need to have all the types in celestia-openrpc eventually. but for now, we can have only what is needed.

Reply:
`header` package was not introduced in this PR, although some helper methods were added. Regardless, depending on whether we want to use `SharesAvailable` for `CheckBlockAvailability`, this can be removed.

Reply:
Created issue for tracking, assuming this is not a blocker: https://github.com/rollkit/celestia-openrpc/issues/19